buildpack.md

```diff
@@ -372,10 +374,13 @@ For each image extension ("extension") or component buildpack ("buildpack") in e
 The selected group MUST be filtered to only include extensions and buildpacks with exit status zero.
 The order of the extensions and buildpacks in the group MUST otherwise be preserved.
 
+For each `/bin/detect` executable in each buildpack, the lifecycle:
```

Comment:
```suggestion
For each `/bin/detect` executable in each extension or buildpack, the lifecycle:
```

Reply:
But TBH I don't know if this statement belongs here - we don't have it for user-provided env vars, for example.